cores/arduino/UART.h

```diff
@@ -21,12 +21,16 @@
   Modified 3 December 2013 by Matthijs Kooijman
 */
 
-#ifndef HardwareSerial_h
-#define HardwareSerial_h
+#ifndef _UART_H_
+#define _UART_H_
 
+#include "Arduino.h"
 #include <inttypes.h>
+#include "api/HardwareSerial.h"
 
-#include "Stream.h"
+#if ARDUINO_API_VERSION > 10000
+using namespace arduino;
```

Comment:
Maybe add a comment to explain why this 'using namespace' statement is needed. It us contrary to the usual recommendation of avoiding 'using namespace` in a C++ header file: https://stackoverflow.com/questions/5849457/using-namespace-in-c-headers